src/Microsoft.ML.Transforms/MissingValueReplacing.cs

```diff
@@ -888,6 +886,26 @@ private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, ColInfo info, string src
         }
     }
 
+    /// <summary>
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="MissingValueReplacingTransformer"/>.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | Vector or scalar of floats or doubles |
+    /// | Output column data type | The same as the data type in the input column |
+    /// The resulting <xref:Microsoft.ML.Transforms.MissingValueReplacingTransformer"/> creates a new column, named as specified in the output column name parameters, and
```

Comment:
>    /// The resulting <xre [](start = 0, length = 26)

please add a new line after the markdown table, otherwise it won't be rendered properly.